### DISCUSSION TITLE
Dinamic folder option in flowise

### DISCUSSION THREAD

**Discussion by shin-bot87** [FlowiseAI/Flowise#1779] [Q&A]
Hello,

I've got a flow wich take pdf files from a folder previously defined as a parameter in the model. I would like to know if it's possible modify the folder output from my code. At this way i could choose where to save the files. Must know that i'm using HTML embebbed in a python project with django as a framework.

Thanks in advance!

kind regards.

![flowise-pic](https://github.com/FlowiseAI/Flowise/assets/55994803/05cb1be0-6dd2-43eb-b25c-0001b037a958)

**Comment by cafepoetica** [comment]
I have the same problem...can someone help us? thanks.
